content/en/docs/reference/issues-security/official-cve-feed.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Official CVE Feed
+weight: 25
+outputs:
+  - json
+  - html 
+layout: official-cve-feed
```

Comment:
I recommend _not_ using the same name for the shortcode and the layout. For example, name the layout `cve-feed` instead.

Reply:
Done